.azure-pipelines/azure-pipelines.yml

```
@@ -115,8 +115,8 @@ stages:
           targets:
             - name: CentOS 7
               test: centos7
-            - name: Fedora 37
-              test: fedora37
+            - name: Fedora 38
+              test: fedora38
```

```suggestion
            - name: Fedora 38
              test: fedora38
```
@elpavel should we maybe add this in a separate PR and merge right now? (and the associated line you added to README)
If you agree, feel free to raise a PR

---

i'm not sure if it will work though:)

---

It should work, I will raise the PR for adding Fedora 38 as a test target.

Fedora 38 has packages for both Psycopg 2 and 3, our normal testing setup will use Psycopg 2 and all should work as usual. Psycopg 3 in Fedora 38 is 3.1 which is good for the new code testing as 3.0 doesn't work.

BTW Fedora 36 and 37 also have packages available for Psycopg 3, but the version is 3.0.

---

cool, thanks!

---

Created #527 to cover this